<dec f='llvm/llvm/include/llvm/Support/Errno.h' l='30' type='std::string llvm::sys::StrError(int errnum)'/>
<use f='llvm/llvm/lib/Support/Errno.cpp' l='32' u='c' c='_ZN4llvm3sys8StrErrorEv'/>
<def f='llvm/llvm/lib/Support/Errno.cpp' l='36' ll='72' type='std::string llvm::sys::StrError(int errnum)'/>
<doc f='llvm/llvm/include/llvm/Support/Errno.h' l='29'>/// Like the no-argument version above, but uses \p errnum instead of errno.</doc>
<use f='llvm/llvm/lib/Support/Unix/Unix.h' l='68' u='c' c='_ZL10MakeErrMsgPNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEERKS4_i'/>
<use f='llvm/llvm/lib/Support/Unix/Program.inc' l='408' u='c' c='_ZN4llvm3sys4WaitERKNS0_11ProcessInfoEjbPNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE'/>
